public/javascripts/EarninigsBeforeDepreciationAndTaxesCalculator.js

```diff
@@ -0,0 +1,24 @@
+﻿"use strict";
+
+function $calculate() {
+    if (!isFloat($("#salesRevenue")[0].value) ||
+	    !isFloat($("#rentAndUtil")[0].value) ||
```

Comment:
Please fix your indentation in this file. Your names are better than last time, though.